.storybook/recipes/BaselineCard/BaselineCard.tsx

```diff
@@ -49,64 +58,133 @@ export interface Props {
  */
 export const BaselineCard = ({
   className,
+  linkHref,
+  linkText,
   label,
   body,
   metadata,
   ...other
 }: Props) => {
-  const componentClassName = clsx(styles['baseline-card'], className, {});
+  /**
+   * Initialize refs to use on the card container and the link
+   *  (for clickable card)
+   */
+  const cardRef = useRef() as MutableRefObject<HTMLDivElement>;
+  const linkRef = useRef() as MutableRefObject<HTMLAnchorElement>;
+
+  /**
+   * useEffect hook
+   * 1) On component load or update, check for both a cardRef and a linkRef.
+   * The presence of both means this card has a clickable link.
+   * 2) Attach click handler to the card.
+   * 3) Check to see if text is selected; if so, do not call .click() on the
+   * linkRef
+   * 4) Pass the click to the linkRef.
+   * 5) Cleanup.
+   *
+   * TODO: If the visible text of the card also contains links, these will
+   * need to have an event listener added to them to stop propagation of the
+   * click event; otherwise, users will not be able to click on these inner
+   * links.
```

Comment:
Can you add a story with a link and button in it that shows an example of how to get it to work?

Reply:
Or should we add a follow-up task for that?

Reply:
Do we know if it's a scenario that will likely occur (additional links within a clickable card)? I hesitated to add it b/c I felt like the recipe was getting a tad complex already.

Reply:
I just looked around the Learning Platform product and the cards are all pretty simple, so maybe you could skip this and just put in the component comment that when the card itself is clickable and navigates to another page, it does not support links or buttons in the content of the card.

Reply:
And we should also tell design to add guidance for other designers to not put links or buttons in a clickable card.